metakube/resource_cluster_test.go

```diff
@@ -367,6 +401,11 @@ resource "metakube_cluster" "acctest_cluster" {
 		}
 		services_cidr = "10.240.16.0/18"
 		pods_cidr = "172.25.0.0/18"
+		{{if .CNIPlugin}}
+		cni_plugin {
+			type = "{{ .CNIPlugin }}"
+		  }
+		{{end}}
```

Comment:
Just keep the usual whitespace:
```suggestion
		{{ if .CNIPlugin }}
		cni_plugin {
			type = "{{ .CNIPlugin }}"
		  }
		{{ end }}
```